pipeline-model-definition/src/test/resources/envDollarQuotes.groovy

```diff
@@ -39,10 +39,10 @@ pipeline {
             }
 
             steps {
-                sh 'echo "FOO is $FOO"'
-                sh 'echo "BAR is $BAR"'
-                sh 'echo "BAZ is $BAZ"'
-                sh 'echo "SPLODE is $SPLODE"'
+                echo "FOO is $FOO"
```

Comment:
This is not really the same. In the former version you tested variable expansion in `sh` this one tests it in groovy-cps.

Reply:
But that probably doesn't matter.

Reply:
Very good point. @jtnord or anyone else - how would I write to a file from `bat`?

Reply:
`echo FOO is %FOO% > somefile.txt`

Reply:
Okiedokie.

Reply:
Oh wait, that's for a different test. Ehhhh, I'll update 'em both.